src/pages/StoryPage.tsx

```diff
@@ -33,8 +34,21 @@ export const StoryPage: React.FunctionComponent<{ nonce?: string }> = ({
             )
           }
 
+          if (block.component === 'footer_block_brand_pivot') {
```

Comment:
Instead of using this if i think we could just remove it from the map of block names/blocks right? I feel like that would be more clean.

Reply:
Much better!